po/sv_SE.po

```diff
@@ -0,0 +1,247 @@
+#: src/apprt/gtk/ui/1.0/menu-window-titlebar_menu.blp:18
+msgid "New Window"
+msgstr "Nytt fönster"
+
+#: src/apprt/gtk/ui/1.0/menu-surface-context_menu.blp:81
+#: src/apprt/gtk/ui/1.0/menu-window-titlebar_menu.blp:23
+msgid "Close Window"
+msgstr "Stäng fönster"
+
+#: src/apprt/gtk/ui/1.0/menu-surface-context_menu.blp:89
+msgid "Config"
+msgstr "Konfiguration"
+
+#: src/apprt/gtk/ui/1.0/menu-surface-context_menu.blp:92
+#: src/apprt/gtk/ui/1.0/menu-window-titlebar_menu.blp:90
+msgid "Open Configuration"
+msgstr "Öpnna konfiguration"
+
+#: src/apprt/gtk/ui/1.0/menu-surface-context_menu.blp:97
+#: src/apprt/gtk/ui/1.0/menu-window-titlebar_menu.blp:95
+msgid "Reload Configuration"
+msgstr "Ladda om konfiguration"
+
+#: src/apprt/gtk/ui/1.0/menu-window-titlebar_menu.blp:85
+msgid "Terminal Inspector"
+msgstr "Terminalinspektören"
+
+#: src/apprt/gtk/ui/1.0/menu-window-titlebar_menu.blp:102
+#: src/apprt/gtk/Window.zig:933
+msgid "About Ghostty"
+msgstr "Om Ghostty"
+
+#: src/apprt/gtk/ui/1.0/menu-window-titlebar_menu.blp:107
+msgid "Quit"
+msgstr "Avsluta"
+
+#: src/apprt/gtk/ui/1.5/ccw-osc-52-read.blp:6
+#: src/apprt/gtk/ui/1.5/ccw-osc-52-write.blp:6
+msgid "Authorize Clipboard Access"
+msgstr "Tillåt åtkomst till urklipp"
+
+#: src/apprt/gtk/ui/1.5/ccw-osc-52-read.blp:7
+msgid ""
+"An application is attempting to read from the clipboard. The current "
+"clipboard contents are shown below."
+msgstr ""
+"Ett program försöker läsa från urklipp. Det aktuella "
+"innehållet visas nedan."
+
+#: src/apprt/gtk/ui/1.5/ccw-osc-52-read.blp:10
+#: src/apprt/gtk/ui/1.5/ccw-osc-52-write.blp:10
+msgid "Deny"
+msgstr "Neka"
+
+#: src/apprt/gtk/ui/1.5/ccw-osc-52-read.blp:11
+#: src/apprt/gtk/ui/1.5/ccw-osc-52-write.blp:11
+msgid "Allow"
+msgstr "Tillåt"
+
+#: src/apprt/gtk/ui/1.5/ccw-osc-52-write.blp:7
+msgid ""
+"An application is attempting to write to the clipboard. The current "
+"clipboard contents are shown below."
+msgstr ""
+"Ett program försöker skriva till urklipp. Det aktuella "
+"innehållet visas nedan."
+
+#: src/apprt/gtk/ui/1.5/ccw-paste.blp:6
+msgid "Warning: Potentially Unsafe Paste"
+msgstr "Varning: Potentiellt osäker infogning"
+
+#: src/apprt/gtk/ui/1.5/ccw-paste.blp:7
+msgid ""
+"Pasting this text into the terminal may be dangerous as it looks like some "
+"commands may be executed."
+msgstr ""
+"Att klistra in den här texten i terminalen kan vara farligt eftersom det ser ut som "
+"att kommandon kan utföras."
+
+#: src/apprt/gtk/Window.zig:199
+msgid "View Open Tabs"
+msgstr "Huvudmeny"
+
+#: src/apprt/gtk/Window.zig:219
+msgid "View Open Tabs"
+msgstr "Visa öppna flikar"
+
+#: src/apprt/gtk/Window.zig:265
+msgid ""
+"⚠️ You're running a debug build of Ghostty! Performance will be degraded."
+msgstr ""
+"⚠️ Du kör en felsökningsversion av Ghostty! Prestanda kommer att försämras."
+
+#: src/apprt/gtk/Window.zig:681
+msgid "Reloaded the configuration"
+msgstr "Laddade om konfigurationen"
```

Comment:
```suggestion
msgstr "Uppdaterade inställningar"
```

Reply:
I disagree on this one. "Uppdaterade" sounds like you wrote to the config, which is not the case here, right?

Reply:
I mean... it depends on whether one interprets "updating" as updating the config file or the internal config state of the app. I personally would just avoid that word in order to stop that confusion from even occurring

Reply:
>  "Uppdaterade" sounds like you wrote to the config

Good point. I just find "Laddade om" confusing as well.